packages/utils/telemetry-utils/src/utils.ts

```diff
@@ -16,6 +16,8 @@ import { ITelemetryGenericEventExt, ITelemetryLoggerExt } from "./telemetryTypes
  * @param logger - The logger to log with
  * @param event - The string or event to log
  * @returns The outcome of the condition
+ *
+ * @public
```

Comment:
internal, probably eventually should be moved to core-utils along with `assert`?

Reply:
Tagged as `@internal` for now and added a `@privateRemarks` block noting that we may want to move this to `core-utils` in the future.

Reply:
Actually, per [this discussion](https://github.com/microsoft/FluidFramework/pull/18047/files/6b9bb7315ee1a33bc9e18c0ec6c4d74376c6dbfe#r1396453925), it looks like this is unused outside of tests inside our repo, and not used by our partners. I went ahead and marked it as `@deprecated`.